node_modules/.package-lock.json

```diff
@@ -207,16 +207,67 @@
       "integrity": "sha512-qelS/Ra6sacc4loe/3MSjXNL1dNQ/GjxNHVzuChwMfmk7HuycRLVQN2qNY3XahK+fZc5E2szqQSKUyAF0E+2bg==",
       "dev": true
     },
+    "node_modules/@types/node-fetch": {
```

Comment:
Is this intentional? There are no new files in the `node_modules`

Reply:
No, I removed it in the node_modules but forgot to remove it in the package-lock.json.